Content: Unfettered campaign Act 1 (#6416)
endless-sky/endless-sky: data/hai/unfettered.txt

	label "no license"
	action
		event "unfettered: outcast"
		clear "unfettered: challengers"
	`The local authorities hail you immediately after the destruction of the Alpha ship with a code purple message.`
		to display
			has "unfettered: destroyed auxiliary"
	`The local authorities hail you immediately after the destruction of the duelling ship with a code purple message.`
		to display
			not "unfettered: destroyed auxiliary"
	branch translator
		has "language: Hai"
	`	They speak in Hai but the tone alone is enough to understand what they mean: you have angered them greatly by destroying that ship, and you should leave, or you risk being shot at. You probably will not be offered missions from them in the future.`
		decline
	label translator
	branch "alpha ship"
		has "unfettered: destroyed auxiliary"
	`	"You have disappointed us greatly by breaking the sacred rules of the duels! Think very carefully before you make any more mistakes, or it will be war! We have removed any privileges you had, and have added you to the list of shame. Rest assured that no True Hai will offer you special missions any longer, but we'll allow you to ferry cargo. For now."`
		decline
	label "alpha ship"

> 	`	"How dare you?! This ship was helping us by providing supplies, and you just shoot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first shot, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`

Reviewer: ```suggestion
	`	"How dare you?! This ship was helping us by providing supplies, and you just shoot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first blood, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`
```

event "unfettered: outcast"
	set "unfettered: outcast"
	clear "salary: unfettered"
	planet "Firelode"
		remove outfitter "Mera's Lab"
	planet "Darkcloak"
		remove outfitter "Unfettered"
	government "Hai (Unfettered)"
		"bribe" .04
	"reputation: Hai (Unfettered)" <?= 0

mission "Unfettered: Regrets"
	landing
	invisible
	to offer
		not "license: Unfettered"
		has "Wanderers: Defend Vara Ke'sok: done"
		not "unfettered: outcast"
	on enter "Ehma Ti" "Wah Yoot" "Hi Yahr"
		event "unfettered: outcast"
		dialog
			`The instant you enter the system, you receive a priority message from the local authorities. "<last>, because of your recent involvements with the aliens up north, we have decided to stop offering you missions."`

> 			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you.`

Reviewer: This also, is it too big for the box?

Author: the box extends, I wouldn't worry about it

Reviewer: That’s good to know.

# using <, > so we can have an empty name
# could add some depending on the kind of warlord you are (and SB you choose):
# He-Shi: sneaky, deadly
# Mem-He: fast, ?
# To-or: enduring, honest
substitutions
	"<uhai>" "Triumph of the True Hai"
	"<, unfettered title>" ", monkey"
	"<, unfettered title>" ", space-monkey"
		has "Unfettered Jump Drive 1: offered"
	"<, unfettered title>" ", Captain"
		has "unfettered: beaten the captain"
	"<, unfettered title>" ", Herald"
		has "salary: Unfettered"
	"<, unfettered title>" ", Warlord"
		"salary: Unfettered" > 1000
	"<, unfettered title>" ", High Warlord"
		has "unfettered: high warlord"
	"<, unfettered title>" ", outcast"
		has "unfettered: outcast"

conversation "unfettered outcast"
	branch "no license"
		has "license: Unfettered"
	action
		# replace it by a wortless outfit
		clear "license: Unfettered"
		outfit "Unfettered License"
	label "no license"
	action
		event "unfettered: outcast"
		event "unfettered: challengers leave"

> 	`The local authorities hail you immediately after the destruction of the Alpha ship with a code purple.`

Reviewer 2: ```suggestion
	`The local authorities hail you immediately after the destruction of the Alpha ship with a code purple message.`
```

		to display
			has "unfettered: destroyed auxiliary"

> 	`The local authorities hail you immediately after the destruction of the duelling ship with a code purple.`

Reviewer 2: ```suggestion
	`The local authorities hail you immediately after the destruction of the duelling ship with a code purple message.`
```

		to display
			not "unfettered: destroyed auxiliary"
	branch translator
		has "language: Hai"
	`	They speak in Hai but the tone alone is enough to understand what they mean: you have angered them greatly by destroying that ship, and you should leave, or you risk being shot at. You probably will not be offered missions from them in the future.`
		decline
	label translator
	branch "alpha ship"
		has "unfettered: destroyed auxiliary"
	`	"You have disappointed us greatly by breaking the sacred rules of the duels! Think very carefully before you make any more mistakes, or it will be war! We have removed any privileges you had, and have added you to the list of shame. Rest assured that no True Hai will offer you special missions any longer, but we'll allow you to ferry cargo. For now."`
		decline
	label "alpha ship"

> 	`	"How dare you?! This ship was helping us by providing supplies, and you just shoot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first shot, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`

Reviewer 2: ```suggestion
	`	"How dare you?! This ship was helping us by providing supplies, and you just shot it down? Disabling it would have been more than fine, looting it tolerated, but destroying the source of our independence? We cannot tolerate it. It matters not who may have drawn the first shot, you have hereby lost all privileges, and it will be war if you make the slightest mistake!"`
```

Author: I think both are correct, I didnt mean it just happened I meant you casually shoot it down

event "unfettered: outcast"
	set "unfettered: outcast"
	planet "Firelode"
		remove outfitter "Mera's Lab"
	planet "Darkcloak"
		remove outfitter "Unfettered"
	government "Hai (Unfettered)"
		"bribe" .04
	"reputation: Hai (Unfettered)" <?= 0

mission "Unfettered: Regrets"
	landing
	invisible
	to offer
		not "salary: Unfettered"
		has "Wanderers: Defend Vara Ke'sok: done"
		not "unfettered: outcast"
	on enter "Ehma Ti" "Wah Yoot" "Hi Yahr"
		event "unfettered: outcast"
		dialog
			`The instant you enter the system, you receive a priority message from the local authorities. "<last>, because of your recent involvements with the aliens up north, we have decided to stop offering you important missions."`

> 			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you. You assume that your Unfettered license no longer has any value to them.`

Reviewer 2: ```suggestion
			`	Since this is a recorded message you cannot respond, but even if you could they would probably just ignore you. You assume that your Unfettered license no longer has any value to it.`
```

Author: I meant to the uhai

Isnt it better to just remove the two last words tho?

			`	"<last>. From now on, you are an outcast. Your history has been erased and your exploits forgotten. You did bring us Jump Drives, however, and we will continue to allow you to do so, but we will do without your help for any more advanced missions. Your license is now meaningless, so you may as well toss it.`
			`	"Consider yourself lucky we do not blow you up, in honor of our past. But you will give this ship back, when the time is right."`
				to display
					has "flagship model: Modified Shield Beetle"

mission "Unfettered: Betrayed: Failure"
	landing
	invisible
	deadline
	to offer
		has "Unfettered: Betrayed: active"
		or
			has "midas: tracker relay"
			has "flagship model: Modified Shield Beetle"
	on offer
		fail "Unfettered: Betrayed"

mission "Unfettered: Restitution"
	invisible
	to offer
		has "unfettered: outcast"
		has "ship model (all): Modified Shield Beetle"
		has "Wanderers: Alpha Surveillance D: done"
		or
			has "flagship planet attribute: shipyard"
			"total ships" > 1
	on offer
		take ship "Modified Shield Beetle"
			id "unfettered: personal beetle"
			"with outfits"
			"unconstrained"
		conversation

> 			`As soon as you exit your ship to go to the spaceport, it flies off. Confused, you hail your first mate.`

Reviewer 2: This line could use some more work.

Author: ```suggestion
			`The exit hatch suddenly closes behind you as soon as you exit your ship, and you can hear the repulsors coming to full power!  You hail your first mate to demand an explanation.`
```

				to display
					has "flagship model: Modified Shield Beetle"
				goto message

> 			`As you exit your flagship, you notice one of your escorts powers all engines on without any of the usual checks or security measures. You hail the Captain in charge of that ship.`

Reviewer 2: Aren't you missing a goto?

			`Following on a hunch, you routinely check your ships before going to the spaceport. But it is quickly apparent that something is missing: the Modified Shield Beetle. It looks like the Unfettered were not kidding about getting the ship back at any cost."`
				to display
					not "ship model: Modified Shield Beetle"
					has "ship model (all): Modified Shield Beetle"
				decline
			label message

> 			`	They respond, in a state of panick, "I swear I did not order takeoff, Captain! I have no idea what is going on, it's as if the ship is flying itself. We'll head to the escape pods at once and abandon it. I'm sorry, there's nothing we can do, the engineer tells me this is a command that has been in the ship all along: return home."`

Reviewer 2: ```suggestion
			`	They respond, in a state of panic, "I swear I did not order takeoff, Captain! I have no idea what is going on, it's as if the ship is flying itself. We'll head to the escape pods at once and abandon it. I'm sorry, there's nothing we can do, the engineer tells me this is a command that has been in the ship all along: return home."`
```

phrase "Essential Unfettered Mission Fail If Killed"
	word
		`If you did not kill the duelling ship, you may try this mission again later. Otherwise, you have failed an essential Unfettered mission. You may want to reload your previous save if you want to be able to finish the campaign.`

# using <, > so we can have an empty name
# could add some depending on the kind of warlord you are (and SB you choose):
# He'Shi: sneaky, deadly
# Mem'he: fast, ?
# To'or: enduring, honest
substitutions
	"<uhai>" "Triumph of the True Hai"
	"<, unfettered title>" ", monkey"
	"<, unfettered title>" ", space-monkey"
		has "Unfettered Jump Drive 1: offered"
	"<, unfettered title>" ", Captain"
		has "unfettered: beaten the captain"
	"<, unfettered title>" ", Herald"
		has "salary: Unfettered"
	"<, unfettered title>" ", Warlord"
		"salary: Unfettered" > 1000
	"<, unfettered title>" ", High Warlord"
		has "unfettered: high warlord"
	"<, unfettered title>" ", outcast"
		has "unfettered: outcast"

conversation "unfettered outcast"
	branch "no license"
		has "license: Unfettered"
	action

> 		# replace it by a wortless outfit

Reviewer 2: ```suggestion
		# replace it by a worthless outfit
```

			`	"Consider yourself lucky we do not blow you up, in honor of our past. But you will give this ship back, when the time is right."`
				to display
					has "flagship model: Modified Shield Beetle"

mission "Unfettered: Betrayed: Failure"
	landing
	invisible
	deadline
	to offer
		has "Unfettered: Betrayed: active"
		or
			has "midas: tracker relay"
			has "flagship model: Modified Shield Beetle"
	on offer
		fail "Unfettered: Betrayed"

mission "Unfettered: Restitution"
	invisible
	to offer
		has "unfettered: outcast"
		has "ship model (all): Modified Shield Beetle"
		has "Wanderers: Alpha Surveillance D: done"
		or
			has "flagship planet attribute: shipyard"
			"total ships" > 1
	on offer
		take ship "Modified Shield Beetle"
			id "unfettered: personal beetle"
			"with outfits"
			"unconstrained"
		conversation
			`The exit hatch suddenly closes behind you as soon as you exit your ship, and you can hear the repulsors coming to full power! You hail your first mate to demand an explanation.`
				to display
					has "flagship model: Modified Shield Beetle"
				goto message

> 			`As you exit your flagship, you notice one of your escorts powers all engines on without any of the usual checks or security measures. You hail the Captain in charge of that ship.`

Reviewer: ```suggestion
			`As you exit your flagship, you notice one of your escorts powers all engines on without any of the usual checks or safety measures. You hail the captain in charge of that ship.`
```

				goto message
			`Following on a hunch, you routinely check your ships before going to the spaceport. But it is quickly apparent that something is missing: the Modified Shield Beetle. It looks like the Unfettered were not kidding about getting the ship back at any cost."`
				to display
					not "ship model: Modified Shield Beetle"
					has "ship model (all): Modified Shield Beetle"
				decline
			label message

> 			`	They respond, in a state of panic, "I swear I did not order takeoff, Captain! I have no idea what is going on, it's as if the ship is flying itself. We'll head to the escape pods at once and abandon it. I'm sorry, there's nothing we can do, the engineer tells me this is a command that has been in the ship all along: return home."`

Reviewer: ```suggestion
			`	They respond, in a state of panic, "I swear I did not order takeoff, Captain! I have no idea what is going on, it's as if the ship is flying itself. We'll head to the escape pods at once and abandon it. I'm sorry, there's nothing we can do. The engineer tells me this is a command that has been in the ship all along: return home."`
```
